lsst-epo/rubin-obs-client
# [F] PublicationsList Content Block

**Related to**
https://github.com/lsst-epo/rubin-obs-api/pull/230

**What it Does**
Adds a Publication Entry type and a Publication List Content Block available for page entries

**How to Test**
1. Create a handful of Publications, some with optional fields filled in, some without.
2. Add a PublicationsList Content Block to a page
3. Save page entry and view on the frontend.  Expect no content block to render if no Publications have been added to the PublicationsList Content Block
4. Add some Publications to the PublicationsList Content Block
5. Save page entry and view on the frontend.  Expect content block to render now that it has Publications added
